dashboard/client/package.json

```diff
@@ -53,6 +48,11 @@
   "resolutions": {
     "@types/react": "16.9.26"
   },
+  "jest": {
+    "transformIgnorePatterns": [
```

Comment:
The new version of lowlight has extended its syntax to TypeScript, which requires additional configuration when using Jest for testing. The official recommendation is to adapt to the following scheme:
* If you are trying to use TypeScript, see https://jestjs.io/docs/getting-started#using-typescript
* To have some of your "node_modules" files transformed, you can specify a custom "transformIgnorePatterns" in your config.

We are currently adding configuration here, is it suitable?

Reply:
CI will fail, the error message:

> Jest failed to parse a file. This happens e.g. when your code or its dependencies use non-standard JavaScript syntax, or when Jest is not configured to support such syntax.
> 
>     Out of the box Jest supports Babel, which will be used to transform your files into valid JS based on your Babel configuration.
> 
>     By default "node_modules" folder is ignored by transformers.
> 
>     Here's what you can do:
>      • If you are trying to use ECMAScript Modules, see https://jestjs.io/docs/ecmascript-modules for how to enable it.
>      • If you are trying to use TypeScript, see https://jestjs.io/docs/getting-started#using-typescript
>      • To have some of your "node_modules" files transformed, you can specify a custom "transformIgnorePatterns" in your config.
>      • If you need a custom transformation specify a "transform" option in your config.
>      • If you simply want to mock your non-JS modules (e.g. binary assets) you can stub them out with the "moduleNameMapper" config option.
> 
>     You'll find more details and examples of these config options in the docs:
>     https://jestjs.io/docs/configuration
>     For information about custom transformations, see:
>     https://jestjs.io/docs/code-transformation